### ISSUE TITLE
Using ggplotwithyourdata R package

### ISSUE THREAD

**Issue by smouksassi** [isop-phmx/studyGroup#29]
- General Topic: An introduction to ggplotwithyourdata Open Source Shiny application
- Youtube link: https://www.youtube.com/watch?v=1rBBmJUIZhs
- Presenter(s): Samer Mouksassi
- Date: June 19, 2017
- Location: Online
- Time: 12:00 to 13:00 EST
- Background Resources for the Audience: https://github.com/smouksassi/ggplotwithyourdata 

Learning outcome:
Being able to graphically explore csv files, generate insights and outlines findings.
Being able to rearrange/edit the graphs for better communication

**Comment by smouksassi** [comment]
Hi Vijay, proposing to do this the week of June 19th after PAGE and after my Home Week in Seattle bests, SM

**Comment by vjd** [comment]
@smouksassi perfect. I will get it done. Can you also ask Ben Rich if he is interested in presenting table1?

**Comment by smouksassi** [comment]
@benjaminrich are you available on that week ?

**Comment by benjaminrich** [comment]
Yes, I will be available that week (but not the following week of June 26th).

**Comment by smouksassi** [comment]
link to youtube event: https://www.youtube.com/watch?v=1rBBmJUIZhs
